### PR TITLE
Improving pt_BR radio/phone number generation

### PR DESCRIPTION
**Pull request by igorsantos07**
Actually any radio number used as a cellphone should have the ninth digit as well.

### CODE REVIEW

File: src/Faker/Provider/pt_BR/PhoneNumber.php [hunk 1]
```diff
@@ -33,14 +33,13 @@ public static function areaCode()
      * Generates a 8/9-digit cellphone number without formatting characters.
      * @param bool $formatted [def: true] If it should return a formatted number or not.
      * @param bool $ninth     [def: false] If the number should have a nine in the beginning or not.
-     *                        If the generated number begins with 7 this is ignored.
```
Comment (fzaninotto) [inline src/Faker/Provider/pt_BR/PhoneNumber.php]: Can you link to a public explanation of the rule behind that logic?

Comment (igorsantos07) [inline src/Faker/Provider/pt_BR/PhoneNumber.php]: It's explained in the docs for `$ninthDigitAreaCodes`. In short, Brazilian cellphones are receiving a ninth digit in a phased way, across our area codes.

**Comment by fzaninotto** [comment]
needs rebase

**Comment by igorsantos07** [comment]
Actually #623 seems like an improvement over this. I should have closed this PR, sorry!

**Pull request closed without merging**